src/color-palette.test.js

```diff
@@ -1,10 +1,45 @@
 import React from 'react'
-import ReactDOM from 'react-dom'
+import { mount } from 'enzyme'
 import ColorPalette from './color-palette'
 
-test('should render a paragraph if there are no colors', () => {
-  const div = document.createElement('div')
-  ReactDOM.render(<ColorPalette />, div)
-  expect(div.querySelectorAll('p')).toBeDefined()
-  ReactDOM.unmountComponentAtNode(div)
+test('that ColorPalette renders an instruction if there are no colors', () => {
```

Comment:
Any reason why we use the `it` alias in `App.test.js` and the `test` alias here?
fwiw I prefer `test`